easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.10.9-gompi-2021a.eb

```diff
@@ -0,0 +1,104 @@
+easyblock = 'Bundle'
+
+name = 'SRA-Toolkit'
+version = '2.10.9'
```

Comment:
Why not upgrade to a newer version?
https://github.com/ncbi/sra-tools/tags

Reply:
I've tried 3.0.0, 2.11.3 and 2.11.2 but I do not manage to build them.